Utils.py

```diff
@@ -217,8 +217,11 @@ def get_public_ipv6() -> str:
     return ip
 
 
+OptionType = typing.Dict[str, typing.Dict[str, typing.Any]]
```

Comment:
should this maybe be called `OptionsType`? Singular sounds confusing in this context.